auto_nag/scripts/good_first_bug_unassign_inactive.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+
+class GoodFirstBugUnassignInactive(BzCleaner):
+    def __init__(self):
+        super(GoodFirstBugUnassignInactive, self).__init__()
+        self.default_assignee = utils.get_config(self.name(), "default_assignee")
+        self.nmonths = utils.get_config(self.name(), "months_lookup")
+
+        self.autofix_assignee = {}
+
+    def description(self):
+        return "Bugs with good-first-bug keyword and no activity for the last {} months".format(
+            self.nmonths
+        )
+
+    def get_autofix_change(self):
+        return self.autofix_assignee
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        doc = self.get_documentation()
+
+        self.autofix_assignee[bugid] = {
+            "comment": {
+                "body": "This good-first-bug hasn't had any activity for {} months, it is automatically unassigned.\n{}".format(
+                    self.nmonths, doc
+                )
+            },
+            "assignee": self.default_assignee,
+            "status": "NEW",
+        }
+
+        return bug
+
+    def get_bz_params(self, date):
+        fields = ["assigned_to"]
+        params = {
+            "include_fields": fields,
+            "resolution": "---",
+            "f1": "keywords",
+            "o1": "casesubstring",
+            "v1": "good-first-bug",
+            "f2": "assigned_to",
+            "o2": "notequals",
+            "v2": self.default_assignee,
```

Comment:
Some components don't use this default_assignee: you could use:
https://github.com/mozilla/relman-auto-nag/blob/71f7f87a6e618bd63b15a026050f6656b49998d4/auto_nag/utils.py#L99

Reply:
This is a bit tricky for autofix. Does `"assigned_to": None` work? Or is there a function to reset the assignee?

Reply:
good question: I don't kwow. 
You can try to make queries here: https://bugzilla-dev.allizom.org/home to check your hypothesis or ask to either glob or dkl.
My way to test with allizom:
```python
from libmozdata.bugzilla import Bugzilla

Bugzilla.URL = 'https://bugzilla-dev.allizom.org'
Bugzilla.API_URL = Bugzilla.URL + '/rest/bug'
Bugzilla.TOKEN = 'your token https://bugzilla-dev.allizom.org'

bug_id = '1395057'

data = {'assignee': ''}
Bugzilla(bugids=[bug_id]).put(data)
```

Reply:
Solved, see: https://bmo.readthedocs.io/en/latest/api/core/v1/bug.html#update-bug

`reset_assigned_to` is the right way.